src/main/scala/com/gd/twitteranalytics/StreamingTweetsJob.scala

```diff
@@ -7,47 +7,37 @@ import org.apache.spark.SparkConf
 import org.apache.spark.streaming.dstream.DStream
 import org.apache.spark.streaming.{Seconds, StreamingContext}
 import java.util.{Date => JavaDate}
-
 import com.gd.twitteranalytics.util.AppConfigReader
-import com.typesafe.config.ConfigException
 
 object StreamingTweetsJob {
 
   val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
 
   def main(args: Array[String]): Unit = {
-    try {
       val authKeys = ConfigValidator.validateTwitterAuth
       if (ConfigValidator.isConfValid(authKeys)) {
-        TweetsDataReader.configureTwitter()
-
         //Incase Only tweets related to a specific Hashtags are needed..
         val filters = AppConfigReader.HashTags
-        val dateColumn = "event_date"
 
+        val dateColumn = "event_date"
         val ssc = setSparkStreamingContext
+
+        TweetsDataReader.configureTwitter
         val englishTweets = TweetsDataReader.getTweets(ssc, Array(filters))
         val tweetsInfo = getDateAndText(englishTweets)
+
         processTweetsInfo(tweetsInfo,AppConfigReader.SavePath,dateColumn)
 
         ssc.start
         ssc.awaitTermination()
       }
       else
         System.exit(1)
-    }
-    catch {
-      case exception: Exception => {
-        handleException(exception)
-      }
-    }
   }
 
   def setSparkStreamingContext() = {
-
-    //TODO: Add Spark Configuration via command line
     val sparkConf = new SparkConf().
-      setAppName(AppConfigReader.AppName).setMaster(AppConfigReader.MasterUrl)
+                        setAppName("twitter-analytics").setMaster(AppConfigReader.MasterUrl)
```

Comment:
there could be multiple spark jobs within the same "twitter-analytics" project, so the project name should not be used for this purpose.

usually, driver class name is used instead (StreamingTweetsJob).
also, now  the driver is responsible for data excitation and loading it to data-lake  (such process sometimes called ETL or Ingress or Collection), so it makes sense to make its name telling about this fact.